### PR TITLE
docs(icons): Replace icon list

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: common-features/icons.md [hunk 1]
```diff
@@ -236,15 +238,18 @@ It is possible to configure the icon type for the whole application:
 
 ## Icons List
 
-Here is a list of all available built-in icons in Telerik UI for Blazor.
+The [Telerik Design System](https://www.telerik.com/design-system/docs/) website provides a list of all [**built-in icons in Telerik UI for Blazor**](https://www.telerik.com/design-system/docs/foundation/iconography/icon-list/).
 
-To define an icon with C# syntax, remove the "k-i-" part and use a PascalCase name. For example, `k-i-caret-tr` will become `FontIcon.CaretTr` or `SvgIcon.CaretTr`.
+To define an icon with C# syntax, replace the **kebap-case** with **PascalCase**. For example, `plus-outline` should become `FontIcon.PlusOutline` or `SvgIcon.PlusOutline`.
```
Comment (radkostanev) [inline common-features/icons.md:243]:
```suggestion
To define an icon with C# syntax, replace the **kebab-case** with **PascalCase**. For example, `plus-outline` should become `FontIcon.PlusOutline` or `SvgIcon.PlusOutline`.
```

Comment (yordan-mitev) [inline common-features/icons.md:243]: Don't use bold for the case styles.
```suggestion
To define an icon with C# syntax, replace the kebab-case with PascalCase. For example, `plus-outline` should become `FontIcon.PlusOutline` or `SvgIcon.PlusOutline`.
```

File: common-features/icons.md [hunk 2]
```diff
@@ -236,15 +238,18 @@ It is possible to configure the icon type for the whole application:
...
 
->caption Built-in Telerik Blazor Icons
+Each icon box in the icon list is clickable and reveals the following details:
 
-Please allow a few seconds for all icons to render. It is also possible to [open the frame as a standalone web page](https://telerik.github.io/kendo-icons/).
+* CSS class aliases, if such exist. Use the CSS classes for [manual HTML rendering of font icons](#render-font-icons-with-html). For example `k-i-plus` is equivalent to `k-i-add`. The C# icon names have no aliases.
```
Comment (yordan-mitev) [inline common-features/icons.md:247]:
```suggestion
* CSS class aliases, if such exist. Use the CSS classes for [manual HTML rendering of font icons](#render-font-icons-with-html). For example, `k-i-plus` is equivalent to `k-i-add`. The C# icon names have no aliases.
```

File: common-features/icons.md [hunk 3]
```diff
@@ -236,15 +238,18 @@ It is possible to configure the icon type for the whole application:
...
+* Unicode representation of the font icon glyph. For example, `\e11e` corresponds to the `plus` icon.
+* Ability to copy the glyph symbol of the font icon.
+* Ability to copy the HTML markup (`<svg>` tag) of the SVG icon.
 
-<iframe src="https://telerik.github.io/kendo-icons/" style="width: 100%; height: 550px;"></iframe>
+The icon list may contain icons, which are not available in older versions of Telerik UI for Blazor, or even in the latest one. Such icons will be added in the next product version.
```
Comment (yordan-mitev) [inline common-features/icons.md:252]:
```suggestion
The icon list may contain icons which are not available in older versions of Telerik UI for Blazor or even in the latest one. Such icons will be added in the next product version.
```

File: common-features/icons.md [hunk 4]
```diff
@@ -236,15 +238,18 @@ It is possible to configure the icon type for the whole application:
 
 ## Icons List
 
-Here is a list of all available built-in icons in Telerik UI for Blazor.
+The [Telerik Design System](https://www.telerik.com/design-system/docs/) website provides a list of all [**built-in icons in Telerik UI for Blazor**](https://www.telerik.com/design-system/docs/foundation/iconography/icon-list/).
 
-To define an icon with C# syntax, remove the "k-i-" part and use a PascalCase name. For example, `k-i-caret-tr` will become `FontIcon.CaretTr` or `SvgIcon.CaretTr`.
+To define an icon with C# syntax, replace the **kebab-case** with **PascalCase**. For example, `plus-outline` should become `FontIcon.PlusOutline` or `SvgIcon.PlusOutline`.
```
Comment (dimodi) [inline common-features/icons.md:243]:
```suggestion
To define an icon with C# syntax, replace the kebab-case with PascalCase. For example, `plus-outline` should become `FontIcon.PlusOutline` or `SvgIcon.PlusOutline`.
```

**Pull request merged**